### PR TITLE
Merges latest changes from 9.4.x into development

### PR DESCRIPTION
**Pull request by valadas**
## Summary
This is a release management task to merge the latest changes from the release/9.4.x branch into the development branch, for this reason we are self approving it as per our release management policies

### CODE REVIEW

File: DNN Platform/Website/DotNetNuke.Website.csproj [hunk 1]
```diff
@@ -0,0 +1,6882 @@
...
+    <Content Include="Resources\Shared\components\CodeEditor\mode\sql\sql.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\stex\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\stex\stex.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\stex\test.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\stylus\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\stylus\stylus.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\swift\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\swift\swift.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\tcl\index.html" />
+
```
Comment (bdukes) [inline DNN Platform/Website/DotNetNuke.Website.csproj]: Looks like this file got duplicated in the merge

Comment (valadas) [reply DNN Platform/Website/DotNetNuke.Website.csproj]: Yep, just resubmitted, let's see...

**Comment by valadas** [comment]
I will tackle the other build issues later tonight...

**Comment by valadas** [comment]
Should be fixed now

**Pull request merged**